<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>350</y>
      <width>421</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>330</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Server Logs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PORT : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_port">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Server IP : </string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_server_state">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>server state :</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_send_client">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>421</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>630</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>broadcasting!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>send message</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_client_info">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>421</width>
      <height>171</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>881</width>
      <height>16777215</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string>socket descriptor</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>IP</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>port</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>connect time</string>
     </property>
    </column>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_server_ip">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_port_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>948</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
